geojson/feature.go

```diff
@@ -9,7 +9,7 @@ import (
 
 // A Feature corresponds to GeoJSON feature object
 type Feature struct {
-	ID         interface{}  `json:"id,omitempty"`
```

Comment:
GeoJSON spec says an id can be a number or a string https://tools.ietf.org/html/rfc7946#section-3.2

people use both, I've been burned by this before https://github.com/paulmach/go.geojson/pull/4

Reply:
Good point, I didn't realize this was specified in the GeoJSON RFC, I've only been using the [MapBox Vector Tile spec](https://github.com/mapbox/vector-tile-spec).